### DISCUSSION TITLE
Changing companies.  Is there a way to transfer ownership of drawings?

### DISCUSSION THREAD

**Discussion by jfaigan** [jgraph/drawio#4658] [Q&A] [answered]
Hello all,

My division was just sold to another company and we need to transfer our documents.  I have many drawings associated with my work Google Drive and work email.  I was wondering if there is an easy way to transfer these diagrams to my new email account?

Thanks for any help.

Joel

**Comment by davidjgraph** [comment]
We don't have any access to your Google Drive. Isn't your whole Google Drive contents going to be transferred anyway?

**Reply by jfaigan** [reply, marked as answer]
Yes, it will, however, not to a Google Drive.  The contents will be zipped and transferred. The new company uses SharePoint.  

My concern was that there was a link between my Google Drive account and the diagram.  I was concerned that I would no longer be able to open the file as my user would no longer exist.

After some investigation, I believe it will not be a problem.  I can always download the transferred file to the desktop and use the desktop version of the application.
